src/uct/ib/efa/srd/srd_ep.c

```diff
@@ -48,6 +49,20 @@ static UCS_CLASS_INIT_FUNC(uct_srd_ep_t, const uct_ep_params_t *params)
 
     self->dest_qpn = uct_ib_unpack_uint24(if_addr->qp_num);
 
+    status = uct_srd_iface_add_ep(iface, self);
```

Comment:
can we keep track only of outstanding requests , and not all endpoint, like we do with flow control?
BTW, can we unite this with FC code? it's very similar to sending "hard request". if we got "pure grant" back, it means the other side created ah.